index.js

```diff
@@ -142,7 +200,18 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  let highest = movies[0].boxOffice
+  for (let movie of movies){
+    if(movie.boxOffice > highest){
+      highest = movie.title;
+    }  if(!movies){
```

Comment:
So close here!  What happens in the event that the _first movie_ is actually the largest box office?  What would we return and why?  Also, remember - an empty array is not a falsy value!  What can we add to 208 to account for an empty array?  Also we should validate this check outside of our for loop so it only happens once..